rust/src/filetracker.rs

```diff
@@ -28,6 +28,8 @@
  * The tracker does continue to follow the file.
  */
 
+//! Gap handling and Chunk-based file transfer tracker.
+
```

Comment:
`cargo clippy` isn't happy with this doc comment (cf error[E0753]: expected outer doc comment). I'm not sure why, but I'm wondering if that's because we have a previous comment before. 

I would suggest either removing this, as this module has a description, already, or moving this doc comment to before the file and author comment...